.emacs.d/init.el

```diff
@@ -1,5 +1,8 @@
 ;;; dot/.emacs
 
+;; load any absolute must-have code
+(load "~/.emacs-emergency" 'noerror 'nomessage 'nosuffix)
```

Comment:
I don't know if `~/.emacs-emergency` is the best name.  I like the location, but something more like `~/.emacs-preload` would be a more descriptive name.  Would that work?

Reply:
Yeah, that seems fine.